client/src/pages/module.js

```diff
@@ -30,7 +31,8 @@ const GET_MODULE_AND_PARENT_TRACK = gql`
  * Module page fetches both parent track and module's data from the gql query GET_MODULE_AND_PARENT_TRACK
  * and feeds them to the ModuleDetail component
  */
-const Module = ({ moduleId, trackId }) => {
+const Module = () => {
+  const { moduleId, trackId } = useParams()
```

Comment:
Do you have Prettier or anything installed? This and a couple of other lines are missing semi-colons.